Sources/Core/Client/Channel/Channel+RxEvents.swift

```diff
@@ -48,7 +48,18 @@ public extension Reactive where Base == Channel {
     
     /// An observable channel unread count.
     var unreadCount: Observable<ChannelUnreadCount> {
-        Client.shared.rx.channelUnreadCount(base)
+        .create { [unowned base] (observer) -> Disposable in
+            let subscription = base.subscribeToUnreadCount { (result) in
+                do {
+                    let response = try result.get()
+                    observer.onNext(response)
+                } catch {
+                    observer.onError(error)
+                }
+            }
+            
+            return Disposables.create { subscription.cancel() }
+        }
```

Comment:
Why did you change this?

Reply:
Because we don't have `channelUnreadCount` anymore.

Reply:
How it coupled with `Pagination`? :)

Reply:
Changing pagination caused `channelUnreadCount` to not compile anymore. Instead of fixing it, I just removed it (via discussion) and removing it caused this to fail. So I refactored.